airsonic-main/src/main/java/org/airsonic/player/Application.java

```diff
@@ -56,6 +57,11 @@ public ServletRegistrationBean dwrServletRegistrationBean() {
         return servlet;
     }
 
+    @Bean
+    public TerminateBean terminateBean() {
+        return new TerminateBean();
+    }
+
```

Comment:
I would actually just register this in the xml file for the legacy profile: https://github.com/airsonic/airsonic/blob/master/airsonic-main/src/main/resources/applicationContext-db-legacy.xml.

Reply:
Sounds reasonable, will do that!